helper/chart_data.js

```diff
@@ -2263,3 +2263,225 @@ const createSolutionsArray = async function (data) {
         console.log(err);
     }
 }
+
+
+//Chart data preparation for entity level assessment report
+exports.entityLevelReportChartCreateFunc = async function(data){
+   
+      //group the data based on completed date   
+      let groupedSubmissionData = await groupArrayByGivenField(data, "completedDate");
+
+      let completedDate = Object.keys(groupedSubmissionData);
+      
+      //Sort the date in ascending order
+      let sortedDate = completedDate.sort();
+
+      if(completedDate.length > config.druid.no_of_assessment_submissions_threshold) {
```

Comment:
@deepakr123 - Define default value if not provided in config

Reply:
resolved